### PR TITLE
Add TFWConnector implementation in Java

### PR DESCRIPTION
**Pull request by ericnerdo**

### CODE REVIEW

File: java/src/main/java/com/avatao/tfw/tfwconnector/Serializer.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    public static List<String> serializeTFWMessage(ObjectNode message) {
+        return serialize(message);
+    }
+
+    /** 
+     * Creates a multipart message out of a JSON Object.
+     * @param data the JSON object to be serialized.
+     * @return multipart message where the parts are in a List
+     */
+    private static List<String> serialize(ObjectNode data) {
```
Comment (therealkrispet) [inline java/src/main/java/com/avatao/tfw/tfwconnector/Serializer.java]: This method could be `serializeTFWMessage` itself

File: java/src/main/java/com/avatao/tfw/tfwconnector/Serializer.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+ * TFW message format and vice versa.
+ */
+public class Serializer {
+
+    /**
+     * Serializes a JSON message into a multipart TFW message.
+     * @param message message to be serialized for TFW server
+     * @return multipart TFW message
+     */
+    public static List<String> serializeTFWMessage(ObjectNode message) {
```
Comment (therealkrispet) [inline java/src/main/java/com/avatao/tfw/tfwconnector/Serializer.java]: This method is basically pointless (the implementation of `serialize` should just simply go here)

File: java/src/main/java/com/avatao/tfw/tfwconnector/TFWServerConnector.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+import com.avatao.tfw.tfwconnector.Serializer;
+
+/**
+ * Class capable of sending messages to the TFW server and event handlers.
+ */
+public class TFWServerConnector {
+    private ZMQ.Context context;
+    private ZMQ.Socket pushSocket;
+
+    /** URL to the socket responsible for TFW communication. */
```
Comment (therealkrispet) [inline java/src/main/java/com/avatao/tfw/tfwconnector/TFWServerConnector.java]: The variable name properly documents it's purpose

**Review by therealkrispet — CHANGES_REQUESTED** [review]
I like the code quality overall

File: java/src/main/java/com/avatao/tfw/tfwconnector/Serializer.java (hunk 2 above)
Comment (ericnerdo) [reply java/src/main/java/com/avatao/tfw/tfwconnector/Serializer.java]: I merged serialize() into it.

**Review by therealkrispet — APPROVED** [review]
✨🍰✨

**Pull request merged**